api/namex/resources/payment/payment.py

```diff
@@ -308,6 +308,12 @@ def post(self, nr_id, payment_action=NameRequestActions.CREATE.value):
 
                 response = make_response(data, 201)
                 return response
+            # something went wrong with status code above
+            else:
+                # log actual status code
+                print('Error with status code. Actual status code: ' + payment_response.statusCode)
+                # return generic error status to the front end
+                return jsonify(message='Name Request {nr_id} encountered an error'.format(nr_id=nr_id)), 500
```

Comment:
Maybe thor will have thoughts on this, but in lear when we have a payment issue we usually return a 402. Probably good to be consistent here

Reply:
we also usually use current_app.logger.debug() instead of print but doesn't effect anything in this case

Reply:
Thanks! changes made